tests/test_evolution_traversal.py

```
@@ -0,0 +1,115 @@
+        # Mock evolution module
+        mock_module = MagicMock()
+        mock_evo.return_value = mock_module
+
+        # Mock list_pending_patches to return empty list
+        mock_module.__getitem__.return_value = MagicMock()
+
+        yield runs_root, mock_module
+
+def test_apply_patch_traversal_blocked(mock_env):
+    """Test that applying a patch with traversal sequence in ID is BLOCKED."""
+    runs_root, mock_evo = mock_env
```

Ruff (0.14.14)

[warning] Unpacked variable `runs_root` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)